PWGCF/FemtoDream/FemtoDreamV0Selection.h

```diff
@@ -120,25 +116,31 @@ class FemtoDreamV0Selection : public FemtoDreamObjectSelection<float, femtoDream
     }
   }
 
-  /// Helper function to obtain the name of a given selection criterion for consistent naming of the configurables
-  /// \param iSel Track selection variable to be examined
-  /// \param prefix Additional prefix for the name of the configurable
-  /// \param suffix Additional suffix for the name of the configurable
-  static std::string getSelectionName(femtoDreamV0Selection::V0Sel iSel, std::string_view prefix = "", std::string_view suffix = "")
+  /// Helper function to obtain the name of a given selection criterion for
+  /// consistent naming of the configurables \param iSel Track selection
+  /// variable to be examined \param prefix Additional prefix for the name of
+  /// the configurable \param suffix Additional suffix for the name of the
+  /// configurable
```

Comment:
Please consider the doxygen related above comments

Reply:
fixed.